quelpa.el

```diff
@@ -142,8 +148,8 @@ If nil the update is disabled and the repo is only updated on
 
 (defcustom quelpa-self-upgrade-p t
   "If non-nil upgrade quelpa itself when doing a
-  `quelpa-upgrade', otherwise only upgrade the packages in the
-  quelpa cache."
+`quelpa-upgrade', otherwise only upgrade the packages in the
```

Comment:
For future reference, with the other upcoming changes, this will need to be changed to `quelpa-upgrade-all` when they are merged, right?

Reply:
Ah, true.